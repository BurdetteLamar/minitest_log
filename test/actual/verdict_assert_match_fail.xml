<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_assert_match?' type='fail'>
    <section_ name='Args=[/b/, &quot;xyz&quot;, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_match?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Regexp' value='/b/'/>
        <actual_ class='String' value='&quot;xyz&quot;'/>
        <exception_ class='Minitest::Assertion' message='Expected /b/ to match # encoding: UTF-8'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[/./, &quot;&quot;, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_match?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Regexp' value='/./'/>
        <actual_ class='String' value='&quot;&quot;'/>
        <exception_ class='Minitest::Assertion' message='Expected /./ to match # encoding: UTF-8'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
