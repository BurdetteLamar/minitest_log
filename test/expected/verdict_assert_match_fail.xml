<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_match?' type='fail'>
    <section name='Args=[/b/, &quot;xyz&quot;]'>
      <verdict method='verdict_assert_match?' outcome='failed' id='0' message='Message 0'>
        <expected class='Regexp' value='/b/'/>
        <actual class='String' value='&quot;xyz&quot;'/>
        <exception class='Minitest::Assertion' message='Expected /b/ to match # encoding: UTF-8'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[/./, &quot;&quot;]'>
      <verdict method='verdict_assert_match?' outcome='failed' id='1' message='Message 1'>
        <expected class='Regexp' value='/./'/>
        <actual class='String' value='&quot;&quot;'/>
        <exception class='Minitest::Assertion' message='Expected /./ to match # encoding: UTF-8'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
